web/src/routes/RepositoryDetails/Tags/TagsActionsAddTagModal.tsx

```diff
@@ -0,0 +1,74 @@
+import { Button, Modal, ModalVariant, TextInput } from "@patternfly/react-core";
+import { useEffect, useState } from "react";
+import { AlertVariant } from "src/atoms/AlertState";
+import { useAlerts } from "src/hooks/UseAlerts";
+import { useTags } from "src/hooks/UseTags";
+
+
+export default function AddTagModal(props: AddTagModalProps){
+    const [value, setValue] = useState('');
+    const {addAlert} = useAlerts();
+    const {createTag, successCreateTag, errorCreateTag} = useTags(props.org, props.repo);
+
+    useEffect(() => { 
+        if(successCreateTag){
+            addAlert({variant: AlertVariant.Success, title: `Successfully created tag ${value}`});
+            setValue('');
+            props.loadTags();
+            props.setIsOpen(false);
+        }
+    }, [successCreateTag]);
+
+    useEffect(() => { 
+        if(errorCreateTag){
+            addAlert({variant: AlertVariant.Failure, title: `Could not create tag ${value}`});
+            setValue('')
+            props.setIsOpen(false)
+        }
+    }, [errorCreateTag]);
+
+    return (
+        <>
+            <Modal
+                id="add-tag-modal"
+                title={`Add tag to manifest ${props.manifest}`}
```

Comment:
could you make the font smaller so that the `sha` doesn't cut off?  The mocks seem to have smaller font size

Reply:
done